src/dev/flang/be/jvm/JVM.java

```diff
@@ -490,18 +491,50 @@ boolean condition(JVM jvm)
       }
       void prepare(JVM jvm)
       {
-        if (!Files.exists(PATH_FOR_CLASSES))
+        try
           {
-            try
+            var m = new Manifest();
+            m.getMainAttributes().put(Attributes.Name.MANIFEST_VERSION, "1.0");
+            m.getMainAttributes().put(Attributes.Name.MAIN_CLASS, "fzC_universe");
+
+            jvm._jos = new JarOutputStream(new FileOutputStream(jvm._fuir.clazzBaseName(jvm._fuir.mainClazzId()) + ".jar"), m);
+          }
+        catch (IOException io)
+          {
+            Errors.error("JVM backend I/O error",
+                         "While creating JAR output stream, received I/O error '" + io + "'");
+          }
+
+        try
+          {
+            String[] dependencies = {
+              "dev/flang/be/interpreter/OpenResources.class",
+              "dev/flang/be/jvm/runtime/Any.class",
+              "dev/flang/be/jvm/runtime/AnyI.class",
+              "dev/flang/be/jvm/runtime/Intrinsics.class",
+              "dev/flang/be/jvm/runtime/Runtime.class",
+              "dev/flang/be/jvm/runtime/Runtime$1.class",
+              "dev/flang/be/jvm/runtime/Runtime$2.class",
+              "dev/flang/be/jvm/runtime/Runtime$Abort.class",
+              "dev/flang/util/ANY.class",
+              "dev/flang/util/List.class",
+            };
+
+            for (var d : dependencies)
               {
-                Files.createDirectory(PATH_FOR_CLASSES);
-              }
-            catch (IOException io)
-              {
-                Errors.error("JVM backend I/O error",
-                             "While creating directory '" + PATH_FOR_CLASSES + "', received I/O error '" + io + "'");
+                jvm._jos.putNextEntry(new JarEntry(d));
+                jvm._jos.write(Files.readAllBytes(jvm._options.fuzionHome()
+                                                              .resolve("classes")
+                                                              .resolve(d)
+                                                              .normalize()
+                                                              .toAbsolutePath()));
```

Comment:
This is ok for now, but eventually I would like to to have a `build/jars/fzrt.jar` created by Makefile that contains these classes and add a dependency in the generated `.jar` file instead of duplicating the classes.